Adding Display Library Web Page

chrisdedman opened the pull request.

chrisdedman committed:
Added images needed for the display

chrisdedman committed:
added styling for display

chrisdedman committed:
init tentative to display library

PenguinBoi12 commented on index.html:
```diff
@@ -0,0 +1,171 @@
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+  <!-- Firebase -->
+  <script src="https://www.gstatic.com/firebasejs/ui/9.6.2/firebase-ui-auth.js"></script>
```
I don't think we'll use Firebase for this project :)

PenguinBoi12 commented on index.html:
```diff
@@ -0,0 +1,171 @@
+        }
+      }
+
+      function mergeTables(headerTablePairs) {
```
Doesn't seem used

PenguinBoi12 commented on index.html:
```diff
@@ -0,0 +1,171 @@
+      }
+
+      function initializeDataTables() {
+        $(document).ready(function () {
```
If you call `initializeDataTables` after the document is parsed you don't need the ready part.

PenguinBoi12 committed:
Create CNAME

PenguinBoi12 committed:
Delete CNAME

PenguinBoi12 committed:
Create CNAME

chrisdedman committed:
overrided dataTables style for mobile

chrisdedman committed:
Refactor: Added modularity to the code + Added table-responsive to the content

chrisdedman committed:
style: update media query for mobile view

chrisdedman committed:
Reverse: update media query for mobile view

chrisdedman committed:
Update SVG dimensions for better rendering on different devices

chrisdedman committed:
chore: Improve responsiveness and styling of navigation bar and content

chrisdedman committed:
style: Adjust margin in styles.css for better spacing

PenguinBoi12 committed:
Update styles.css

PenguinBoi12 committed:
Update index.html

PenguinBoi12 committed:
Update index.html

chrisdedman committed:
style: Update table row background colors for better readability

chrisdedman committed:
update function name, and design table

chrisdedman committed:
removing .ready() for document

chrisdedman committed:
Added C book + link to the website

chrisdedman committed:
updated readme

chrisdedman committed:
style: Improve table styling and responsiveness

chrisdedman committed:
Merge branch 'display' of https://github.com/Code-Society-Lab/resources into display

chrisdedman committed:
remove whitespace

chrisdedman committed:
Merge branch 'main' into display

chrisdedman committed:
feat: Add notification for contributing to the library

Add a notification to the library page, encouraging users to contribute by submitting a pull request to the README file on GitHub. The notification is displayed when the page finishes loading and is appended to the GitHub icon. It fades in after 100 milliseconds and automatically disappears after 5 seconds.

chrisdedman committed:
Added notification styling and positioning

chrisdedman committed:
Update library contribution notification message

chrisdedman committed:
Adjust notification message positioning and width.

chrisdedman committed:
style: Update border color in styles.css #333 -> #000

chrisdedman committed:
cleaned up style file

chrisdedman committed:
removed unnecessary code
